JitsiConference.js

```diff
@@ -591,6 +620,62 @@ JitsiConference.prototype._setupNewTrack = function (newTrack) {
 };
 
 /**
+ * Adds loca WebRTC stream to the conference.
+ * @param {MediaStream} stream new stream that will be added.
+ * @param {function} callback callback executed after successful stream addition.
+ * @param {function(error)} errorCallback callback executed if stream addition fail.
+ * @param {object} ssrcInfo object with information about the SSRCs associated with the
+ * stream.
+ * @param {boolean} [dontModifySources] if true _modifySources won't be called.
```

Comment:
can you clarify a bit when this is needed? is the public API of this function supposed to care? Does it make sense to have some internal `_addLocalStream` which allows for that option to be set but not the public one?